tests/s25Main/lua/testLua.cpp

```diff
@@ -843,6 +844,21 @@ BOOST_AUTO_TEST_CASE(onExplored)
     }
 }
 
+BOOST_AUTO_TEST_CASE(CampaignStatusCanBeChangedFromLua)
+{
+    initWorld();
+
+    SETTINGS.campaigns.saveData["campaign_id"] = "110";
+    executeLua("rttr:SetCampaignChapterCompleted('campaign_id', 2)");
+    executeLua("rttr:SetCampaignChapterCompleted('campaign_id', 4)");
+    executeLua("rttr:EnableCampaignChapter('campaign_id', 5)");
+    executeLua("rttr:SetCampaignChapterCompleted('campaign_id', 0)"); // noop - chapters start from 1
+    executeLua("rttr:EnableCampaignChapter('campaign_id', 0)");       // noop - chapters start from 1
+    executeLua("rttr:EnableCampaignChapter('campaign_id', 2)");       // noop - already completed
+    game.executeAICommands();
```

Comment:
What is this line for?

Reply:
Neither this not initWorld are needed, some leftover garbage most likely.